docs/source/index.mdx

```diff
@@ -21,24 +25,52 @@ description: A strongly-typed, caching GraphQL client for iOS, written in Swift
   </ButtonLink>
 </p>
 
-## Benefits
+## Key features
+
+Apollo iOS consists of three primary components:
+
+* [Code generation](#code-generation)
+* A [normalized cache](#the-normalized-cache)
+* The [networking and GraphQL execution layer](#networking-and-graphql-execution)
+
+When used together, these components dramatically improve the process of requesting, consuming, and persisting GraphQL data.
+
+### Code generation
+
+Apollo iOS generates operation objects for your GraphQL operations, providing a type-safe way to construct operations and provide values for input variables in your Swift code.
+
+Each generated operation contains a set of robust, strongly-typed models for its response. These generated models help you access your GraphQL response data in a type-safe and flexible format. You don't need to deal with parsing JSON responses or passing around dictionaries of values that require manual casting. You also don't need to write model types yourself, because models are generated from the GraphQL operations you define.
+
+Because generated response models are operation-specific, they include properties _only_ for the GraphQL fields included in their corresponding operation. This means you can rely on the Swift type checker to flag data access errors at compile time.
+
+In addition to operation-specific models, Apollo iOS generates models for your GraphQL _fragments_. Using these fragment models alongside the operation-specific models enables you to share common models across multiple GraphQL operations.
+
+Models are also generated for your schema's input types, enums, and custom scalars, all of which improve safety and speed when working with GraphQL data in pure Swift code.
+
+### The normalized cache
 
-### Strong typing with codegen
+Apollo iOS normalizes operation results to build a client-side cache of your data, which is updated with every operation you execute. Our normalized cache is built specificially with GraphQL in mind. It de-duplicates GraphQL response data, merging data from separate GraphQL operations into normalized objects so that it can stay up-to-date with your back end with as few operations as possible.
```

Comment:
```suggestion
Apollo iOS normalizes operation results to build a client-side cache of your data, which is updated with every operation you execute. Our normalized cache is built specificially with GraphQL in mind. It de-duplicates GraphQL response data, merging data from separate GraphQL operations into normalized objects so that it can stay up-to-date with your backend with as few operations as possible.
```